### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by bennyscripts** [ghostselfbot/ghost#4] [Announcements]
Welcome to the Github discussions page for Ghost. This is where you will find support, ideas and announcements about Ghost selfbot. Ghost is not dead! I am gradually working on it when I have inspiration and feel bothered to try and work around Discord's stupid STUPID changes. 

If you have any ideas for commands or features please let me know in the ideas channel. If you need any help with Ghost create a thread in the help channel and either me or someone else will help you out. Please feel free to create pull requests with your commands or features.

**Comment by Bub283** [comment]
yo benny its nice to see you back

**Reply by bennyscripts** [reply]
Cheers man!

**Reply by Bub283** [reply]
will there maybe be a ghost discord or na

**Reply by bennyscripts** [reply]
I have no plans for a dedicated discord server but if Ghost becomes bigger again and I start getting more recommendations and suggestions then potentially.

**Reply by bennyscripts** [reply]
> will there maybe be a ghost discord or na

yeah i decided to make one haha 
https://discord.gg/ayz7eYvFsm
